src/Microsoft.TestPlatform.Common/DataCollection/AfterTestRunEndResult.cs

```diff
@@ -36,7 +37,7 @@ private AfterTestRunEndResult()
     /// The metrics.
     /// </param>
     public AfterTestRunEndResult(Collection<AttachmentSet> attachmentSets, IDictionary<string, object> metrics)
-        : this(attachmentSets, new Collection<InvokedDataCollector>(), metrics)
+        : this(attachmentSets, new Collection<InvokedDataCollector>(), metrics, new Collection<TelemetryEvent>())
```

Comment:
Metrics are already passed here which hold telemetry, can that be reused so we don't have to pass two different objects with telemetry?